IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -0,0 +1,67 @@
+### IPFS All Hands Format
+
+![](https://user-images.githubusercontent.com/157609/48067634-0b5f2780-e1d1-11e8-8b67-30d7aa20896b.gif)
+
+## The Dream
+
+For the past couple of months, we have been discussing revamping the IPFS ALL HANDS meetings. If you want more context, please see this issue https://github.com/ipfs/pm/issues/636.
+
+We want the IPFS hands-on meeting to act as a space where we can showcase all of our hard work and game changing projects.The IPFS meetings give viewers a taste of some of the problems we are tackling.
+
+
+## What's in it for you? 😺
+
+The main purpose of the IPFS ALL HANDS CALL is to highlight the most exciting work in the broader IPFS community and disseminate high-value knowledge to the many teams improving IPFS, building on top of it, or doing relevant research. You can use these weekly calls to:
+
+1. Attract skilled contributors to your project
+
+2. Motivate the community about adding support for a new feature
+
+3. Find new partners, capabilities, or challenges for your project to work on/with.
+
+4. Practice speaking in front of a captivated audience
+
+
+## The Format
+
+Every month we will have:
+
+* 15 minute talk from guest presenter (3x per month)
+* 5 minute talks lightening talks (1x per month)
+
+In total we will have a talk every Monday. The times for the talks are:
+
+| Time (PST - UTC -CET ) |
+---|
+9:00 **17:00** 18:00 |
+
+
+If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol labs dot ai.
+
+ ![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1541176158/ipfs_crew_xzbhxr.png)
+
+
+
+
+
+
+## List of Upcoming Talks  ⚡️📅 ⚡️
+![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1542475278/ipfs_calendar_dyxtst.jpg)
+
+
+
+_[Sign up here if you would like to speak!](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898)_
+
+
+
+### While you're here check out some of our past talks 
+
+[IPFS ALL Hands Call November 12, 2018](https://youtu.be/0fT9HC2Crqw)
+
+[IPFS ALL Hands Call November 5, 2018](https://youtu.be/4gVUKbXT0cM)
+
+[IPFS ALL Hands Call October 29, 2018](https://youtu.be/o1tUzgThZK0)
+
+[IPFS ALL Hands Call October 22, 2018](https://youtu.be/iEagJfB8AvM)
```

Comment:
It would be useful if these links would point to the timecode of the start of the talk and also the title was the title of the talk vs the all hands date.

Example

`[Mozilla/libdweb by Irakli Gozalishvili](https://youtu.be/0fT9HC2Crqw?t=132)`


You can get these timecodes by clicking the share button and pinpoint the moment you want it to start:
<img width="1327" alt="image" src="https://user-images.githubusercontent.com/1211152/48694357-671bae80-ebdc-11e8-8ab9-84f50990c85b.png">

Reply:
@pkafei did you noticed that I listed the name of the talk and the speaker vs just saying "IPFS All Hands" in the example above?